package.json

```diff
@@ -18,15 +18,13 @@
     "prelerna-publish": "lerna run build",
     "lerna-publish": "lerna publish from-package --no-push --force-publish --no-verify-access --no-verify-registry",
     "lint": "yarn eslint && yarn stylelint",
-    "eslint": "eslint .",
+    "eslint": "eslint . --ext .js,.jsx,.ts,.tsx",
     "stylelint": "stylelint **/*.scss",
     "test": "lerna run test",
     "commitlint": "commitlint --verbose --edit",
     "prepare": "[ -z \"$CI\" ] && husky install || exit 0"
   },
   "devDependencies": {
-    "eslint": "^7.9.0",
```

Comment:
I've always been a bit confused by what is "best practice" here. So hopefully you can help me understand this a bit better.

Since the `eslint` script is calling an executable, wouldn't we want to keep `eslint` as a dependency in our root `package.json`? Then, I'd expect our embedded package to rely on `eslint` as a peer-dependency, as opposed to relying on it as a `dependency` there.

Reply:
Good question - There is room for creative interpretation here. What we had before works, but it also leads to a proliferation of eslint deps being install at multiple versions which isn't ideal. My aim is to unify all the eslint deps within a single package to minimize duplicate installs as was occurring before.

Also consider that because the monorepo root is not a package itself, having dependencies defined here isn't critical from a development or 3rd party perspective, and we can rely on some amount of monorepo hoisting behavior to make the executable available at this level. An alternative would be a `lerna run` style command that relies on executables being available without hoisting but would also be less performant in the case of the eslint executable here.

`peer-dependency` would make sense if the public API of the components themselves were eslint focused but did not contain any explicit eslint imports. This isn't the case yet - but when we do build our own eslint rules or configuration presets we'll want to consider that approach, or alternatively list our new config only package as a dependency.